components/Navigation.js

```diff
@@ -0,0 +1,45 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+const StyledNav = styled.nav`
+  padding: 1rem;
+  position: absolute;
+  width: 100%;
+`;
+
+const StyledList = styled.ul`
+  list-style: none;
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  justify-content: space-around;
+  position: relative;
+`;
```

Comment:
For better readability,  group related CSS properties together. For example:
display: flex;
justify-content: space-around;
align-items: center;
position: relative;
margins:
padding:
gap: